README.md

```diff
@@ -60,48 +60,23 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 [HELLO-NEW] <a name="about-project">Html/css</a>
 
-> Describe your project in 1 or 2 sentences.
+>  this is the first phase of my first portfolio project
 
-**[your_project__name]** is a...
-
-## 🛠 Built With <a name="built-with"></a>
+## 🛠 Built With <a name="built-with">Html/css</a>
 
 ### Tech Stack <a name="tech-stack"></a>
 
-> Describe the tech stack and include only the relevant sections that apply to your project.
-
-<details>
-  <summary>Client</summary>
-  <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
-  </ul>
-</details>
-
-<details>
-  <summary>Server</summary>
-  <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
-  </ul>
-</details>
-
-<details>
-<summary>Database</summary>
-  <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
-  </ul>
-</details>
+> Html/css
 
-<!-- Features -->
 
-### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
+<!-- Features -->
+
+### Key Features <a name="key-features">hover feature over button</a>
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
```

Comment:
- [ ] Kindly show how you are of your achievement by listing what to look out for in this work e.g. `semantic use of html tags` , `Industry standard for best practices` , etc... 🔧